.github/workflows/lint.yml

```diff
@@ -152,7 +152,7 @@ jobs:
         with:
           cache-on-failure: true
       - uses: taiki-e/install-action@cargo-udeps
-      - run: cargo udeps --workspace --lib --examples --tests --benches --all-features --locked
+      - run: cargo udeps --workspace --lib --examples --tests --benches --all-features --locked --exclude "reth-optimism-*" --exclude op-reth
```

Comment:
because of the `#![cfg(not(feature = "scroll"))]` conditional compilation in the `optimism` crates, we exclude the `udeps` check on the optimism crates.